Makefile

```diff
@@ -148,11 +143,18 @@ cache-era5-gcp:
 		met_vars = ["t", "q", "u", "v", "w", "ciwc", "z", "cc"]; \
 		rad_vars = ["tsr", "ttr"]; \
 		ERA5(time=time, variables=met_vars, pressure_levels=lev, cachestore=cache).download(); \
-		ERA5(time=time, variables=rad_vars, cachestore=cache).download()'
-
+		ERA5(time=time, variables=rad_vars, cachestore=cache).download()' \
+		|| ( \
+		echo -e \
+			"$(COLOR_YELLOW)CDS Credentials required to run notebooks and doc tests\n" \
```

Comment:
Throw the error once here so it breaks out of the tests with a helpful message? Maybe there is a better URL to use here